javalib/src/main/scala/java/security/Timestamp.scala

```diff
@@ -1,7 +1,70 @@
 package java.security
 
+// Ported from Harmony
+
 import java.util.Date
 import java.security.cert.CertPath
 
-final class Timestamp(timestamp: Date, signerCertPath: CertPath)
-    extends Serializable
+import scala.collection.mutable.StringBuilder
+
+/**
+ * Constructs a new instance of {@code Timestamp} with the specified
+ * {@code timestamp} and the given certificate path.
+ *
+ * @param timestamp date and time.
+ * @param signerCertPath the certificate path.
+ * @throws NullPointerException
+ *            if {@code timestamp} is {@code null} or if {@code signerCertPath} is {@code null}.
+ */
```

Comment:
We usually do not include all the javadoc in the classes from the javalib / scalalib, because nobody is ever going to see them.

Reply:
I remove it?

Reply:
@xavier-fernandez I do when I port classes from Harmony, they are not visible anywhere as @Duhemm mentioned.